modules/test/playwright/tests/object-web/objectActions.spec.ts

```diff
@@ -18,51 +18,77 @@ export const test = mergeTests(
 	objectPagesTest
 );
 
+interface CreatedEntities {
+	notificationTemplatesId: number[];
+	objectActionsId: number[];
+	objectDefinition: ObjectDefinition;
+}
+
+const createdEntities = {
+	notificationTemplatesId: [],
+	objectActionsId: [],
+	objectDefinition: {},
+} as CreatedEntities;
```

Comment:
since you are using this `CreatedEntities` interface only one time here you could declare the types directly on the `createdEntities` like this:

```
const createdEntities = {
	notificationTemplatesId: [],
	objectActionsId: [],
	objectDefinition: {},
} as {
	notificationTemplatesId: number[];
	objectActionsId: number[];
	objectDefinition: ObjectDefinition;
};
```
but this is just a comment and a matter of preference, you don't need to change it  :blush: if the other parts of the code in our module are using this pattern you can leave as it is

Reply:
hey @rebsilva, I see! Personally, I agree with your comment, since we have an simple and small object! only maybe if We were to consider the future possibility of it receiving more properties, it would be better to keep this way, probably? 
As you pointed out, the interface is indeed the pattern being used right now through some of our modules, but I would be fine with changing it, as most of them are also like this, with 2/3 properties maximum. Since it's uncertain whether we will have more props or not, we could keep it the way you suggested and then only if needed, we would choose using the interface instead. 
What do you think?

Reply:
i see! you can change it then, and if you think its okay to change in the other files and won't take too much time i think it's a good idea!

Reply:
sure, I will take a look into it right now, thanks!